bottlerocket-settings-models/modeled-types/src/kubernetes.rs

```diff
@@ -20,6 +21,10 @@ use crate::SingleLineString;
 const IMAGE_GC_THRESHOLD_MAX: i32 = 100;
 const IMAGE_GC_THRESHOLD_MIN: i32 = 0;
 
+// Define the bounds for the `time-slicing.replicas` field
+const TIME_SLICING_REPLICAS_MIN: i32 = 2;
+const TIME_SLICING_REPLICAS_MAX: i32 = i32::MAX;
```

Comment:
Does this limit correspond to the `timeSlicing` limit?

Reply:
In the k8-device-plugin code they use an `int`, and verify it is over 2. 

In NVIDIA's technical blog it is marked as unlimited:
https://developer.nvidia.com/blog/improving-gpu-utilization-in-kubernetes/#virtualization_with_vgpu